Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -1228,11 +1228,9 @@ namespace Tensile
         ContractionSolution::TAMetricProblemScore pp,
         ContractionSolution::TAMetricProblemScore ppReference) const
     {
-        double metric = 0.0;
```

Comment:
cppcheck: Variable 'metric' is reassigned a value before the old one has been used.